### PR TITLE
Java 8 oracle

### PR DESCRIPTION
**Pull request by emmanuellyautomated**
add new role for folks provisioning with java 8 (oracle)

### CODE REVIEW

File: java-8-oracle/LICENSE [hunk 1]
```diff
@@ -0,0 +1,674 @@
+GNU GENERAL PUBLIC LICENSE
```
Comment (vforgione) [inline java-8-oracle/LICENSE]: Submodule files should never be checked into the repository

File: java-8-oracle/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+ansible-java8-oracle
```
Comment (vforgione) [inline java-8-oracle/README.md]: ditto

File: java-8-oracle/meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (vforgione) [inline java-8-oracle/meta/main.yml]: ditto

File: java-8-oracle/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (vforgione) [inline java-8-oracle/tasks/main.yml]: ditto

File: java-8-oracle/LICENSE (hunk 1 above)
Comment (emmanuellyautomated) [inline java-8-oracle/LICENSE]: makes sense. I'm not the most experienced with submodules, but I suppose I should add the submodule dir to the `.gitignore`, no?

**Comment by emmanuellyautomated** [comment]
@vforgione I thought it would a universal resource for folks needing to build with java. I was under the impression that the same java install would be used regardless of project.

File: java-8-oracle/LICENSE (hunk 1 above)
Comment (vforgione) [inline java-8-oracle/LICENSE]: You shouldn't have to ignore the submodule files - git is smart enough to only add the reference to `.gitmodules`. How did you add this submodule?

**Comment by vforgione** [comment]
@emmanuellyautomated that makes sense. I was just curious.

File: java-8-oracle/LICENSE (hunk 1 above)
Comment (emmanuellyautomated) [inline java-8-oracle/LICENSE]: gone.

File: java-8-oracle/README.md (hunk 2 above)
Comment (emmanuellyautomated) [inline java-8-oracle/README.md]: gone.

File: java-8-oracle/meta/main.yml (hunk 3 above)
Comment (emmanuellyautomated) [inline java-8-oracle/meta/main.yml]: gone.

File: java-8-oracle/tasks/main.yml (hunk 4 above)
Comment (emmanuellyautomated) [inline java-8-oracle/tasks/main.yml]: gone.

**Comment by vforgione** [comment]
Java: 👎 

Your PR: 👍

**Comment by adamyala** [comment]
👍

**Comment by emmanuellyautomated** [comment]
Ha @vforgione, be thankful I didn't open a PR for a vim ansible role  😆

**Comment by vforgione** [comment]
@emmanuellyautomated ubuntu ships with vim :fire:

**Comment by emmanuellyautomated** [comment]
yeah @vforgione, it does. but the plugins tho.

**Pull request merged**